### DISCUSSION TITLE
Any way to do "group-peer" classes?

### DISCUSSION THREAD

**Discussion by mybearworld** [tailwindlabs/tailwindcss#11642] [Help] [answered]
Take this code: https://play.tailwindcss.com/kIoBGuWG7d
The first block works because the `peer` is right next to the element using it. In the second one, I'd like to use something like `group-peer`, but that doesn't exist. Is there anything I can do?

**Comment by wongjn** [comment, marked as answer]
Consider using an [arbitrary `group-*` variant](https://tailwindcss.com/docs/hover-focus-and-other-states#arbitrary-groups) like `group-[.peer:checked~&]:`: https://play.tailwindcss.com/8rYbJa8Ucl

**Reply by mybearworld** [reply]
Thank you!
